frontend/packages/knative-plugin/integration-tests/features/kafka-event-source/eventing-kafka-event-source.feature

```diff
@@ -5,7 +5,9 @@ Feature: Kafka Event Source
 
         Background:
             Given user has installed Serverless Operator
-              And user has created Knative Kafka instance in knative-eventing namespace
+              And user has created Knative Serving instance in knative-serving namespace
+              And user has created Knative Eventing instance in knative-eventing namespace
```

Comment:
By default this 2 steps are included in Serverless operator installation.
Can we remove them